desktop/src/jvmMain/kotlin/Main.kt

```diff
@@ -0,0 +1,24 @@
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.unit.DpSize
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Window
+import androidx.compose.ui.window.WindowPlacement
+import androidx.compose.ui.window.WindowPosition
+import androidx.compose.ui.window.WindowState
+import androidx.compose.ui.window.application
+import com.xacalet.minesweeper.common.ui.MainScreen
+
+fun main() = application {
+    Window(
+        onCloseRequest = ::exitApplication,
+        title = "Minesweeper",
+        state = WindowState(
+            position = WindowPosition.Aligned(Alignment.Center),
+            size = DpSize(320.dp, 550.dp),
```

Comment:
In classic Minesweeper, the window always adjusted to the size of the board. To replicate the same behaviour, you could set an unspecified size to the `Window` composable. 
```suggestion
            size = DpSize(Dp.Unspecified, Dp.Unspecified),
```